### DISCUSSION TITLE
Qutebrowser and Qt6 on Debian: failure of playing h264 videos.

### DISCUSSION THREAD

**Discussion by jcbroedel** [qutebrowser/qutebrowser#8404] [Q&A] [answered]
Having spent way too much time on trying to get the newest github version of qutebrowser running with my Debian installation (cutting edge Debian testing), I am in need for somebody to help me and to compare with: packages, methods, installation parameters etc..

Here is my method (which worked like a charme before with qt5): 

Install via venv: clone/update the qutebrowser repository, run ./scripts/mkvenv.py --pyqt-type link --pyqt-version 6, run qutebrowser.

I have installed all gstreamer1.0 packages (good,bad,ugly,...) as well as all pyqt6 bindings. No python3-pyside6 - packages. All webengine-related packages (Happily providing a complete list later.) 

Playing h264 videos does not work in falkon either (another browser based on qtwebengine), so it is probably a problem from this side. gst-play-1.0 does play h264 videos perfectly. But could be in the python interface. 

I am aware this is not too much of a qutebrowser problem. However, qutebrowser being my browser for the last three years at least, it is a problem I need to settle.

So: what can I do to debug the problem? The cutebrowser debugging did not help, I couldn't find any codec error or similar. Tried with --temp-basedir as well. 
What else could I try to pin the problem down? Any hints or referrals are very much apreciated.

**Comment by The-Compiler** [comment, marked as answer]
It's a Debian packaging bug in their Qt package, see [#1087829 - qutebrowser: No proprietary codecs suported - Debian Bug report logs](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1087829).

**Reply by jcbroedel** [reply]
Great + thanks! Should have checked Debian but reports myself. Unstable doesn't yet contain a fixed version, will simply revert to previous qt6-webengine.
